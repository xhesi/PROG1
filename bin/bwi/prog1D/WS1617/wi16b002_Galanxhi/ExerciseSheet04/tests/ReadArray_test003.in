1
0
-1
-2
-3
-4
-5
99
66
42
