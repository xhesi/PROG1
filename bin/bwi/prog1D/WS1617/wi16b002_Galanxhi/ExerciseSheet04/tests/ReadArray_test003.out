1. number: 2. number: 3. number: 4. number: 5. number: 6. number: 7. number: 8. number: 9. number: 10. number: Input finished.
[9]: 42
[8]: 66
[7]: 99
[6]: -5
[5]: -4
[4]: -3
[3]: -2
[2]: -1
[1]: 0
[0]: 1
Output finished.